pkg/controller/acmechallenges/sync.go

```diff
@@ -94,18 +92,12 @@ func (c *controller) Sync(ctx context.Context, chOriginal *cmacme.Challenge) (er
 	// challenge.
 	//
 	// API Transition
-	// -- Until UseDomainQualifiedFinalizer is active, we add cmacme.ACMELegacyFinalizer.
-	// -- When it is active we add cmacme.ACMEDomainQualifiedFinalizer instead.
-	//
-	// -- Both finalizers are supported, the flag just controls the one we add.
+	// -- We add cmacme.ACMEDomainQualifiedFinalizer.
+	//    We still support removing cmacme.ACMELegacyFinalizer.
```

Comment:
```suggestion
	//  -- We still support removing cmacme.ACMELegacyFinalizer.
```

Nit: Not sure about the format rules here.

Reply:
I struggled with this (I really did). I'm going to take your form (and skip the attribution if you don't mind)

Reply:
actually. I'm going to remove the reference to the API transition entirely. This code is no longer transitioning.

Reply:
Looking good!